src/index.ts

```diff
@@ -30,9 +30,20 @@ app.use(express.json());
 
 // Hello World endpoint - now renders a view
 app.get("/", (_req, res) => {
+  // Get the 3 most recent jobs for the homepage
+  const allJobs = jobRoleService.getJobRoles();
+  const latestJobs = allJobs
+    .sort((a, b) => b.closingDate.getTime() - a.closingDate.getTime())
+    .slice(0, 3)
```

Comment:
Bit of a magic number, but comment on line 33 documents it so this is alright imo 👍